src/esp/assets/PTexMeshData.cpp

```diff
@@ -5,8 +5,59 @@
 #include "PTexMeshData.h"
 
 #include <fcntl.h>
+#ifndef _WIN32
 #include <sys/mman.h>
 #include <unistd.h>
+#else
+#include <io.h> 
+#include <Windows.h>
+
+#define PROT_READ 1
+#define MAP_PRIVATE 4
+
+// Note: This is not a full implementation of mmap/munmap.  It is is only meant to work in limited scenarios
+void* mmap(void* addr, size_t length, int prot, int flags, int fd, off_t off) {
```

Comment:
@msavva since you wrote the original code -- do you think we could simply use [Utility::Directory::mapRead()](https://doc.magnum.graphics/corrade/namespaceCorrade_1_1Utility_1_1Directory.html#a84c45342a8cec2111853f51873d82c9d) for the memory-mapped data? It's production-tested on Linux, mac/iOS and Windows, so I don't expect any issues by switching to that (except if you use some features which it doesn't have, but doesn't seem so).

Less code in Habitat = less code to worry about :)

Reply:
This is a great suggestion -- it is much preferred to leverage already platform-agnostic code for this from Corrade rather than implementing it here

Reply:
Seconded!

Reply:
Done as a part of #106, this file no longer needs any change to work on Windows.